probe-rs/tests/source_location.rs

```diff
@@ -0,0 +1,101 @@
+use probe_rs::debug::{ColumnType, DebugInfo, SourceLocation};
+use std::path::{Path, PathBuf};
+
+const TEST_DATA: [(u64, u64, ColumnType); 8] = [
+    // Target address, line, column
+    (0x920, 11, ColumnType::LeftEdge),
+    (0x922, 13, ColumnType::Column(21)),
+    (0x92e, 13, ColumnType::Column(11)),
+    (0x938, 27, ColumnType::Column(4)),
+    (0x93a, 14, ColumnType::Column(19)),
+    (0x956, 20, ColumnType::Column(12)),
+    (0x962, 21, ColumnType::Column(12)),
+    (0x96a, 22, ColumnType::Column(12)),
+];
+
+#[test]
+fn breakpoint_location_absolute() {
+    let di = DebugInfo::from_file("tests/gpio_hal_blinky").unwrap();
+
+    // Here we test with an absolute path, i.e. the combination of compilation directory
+    // and relative path to the actual source file.
+    let path = Path::new("/home/dominik/Coding/microbit/examples/gpio_hal_blinky.rs");
```

Comment:
Is this just an example path or is this where the file actually resides? If it's the latter, changing this is appreciated :)

Reply:
The path is hard coded in the debug information. The example file is also in the repository,
so it will not change.

Is there  a reason you want to change it? We need an absolute path here.

Reply:
ah it didn't appear to me that it was required for the example binary. but now that you mention this, it makes sense.

i was just afraid that maybe that maybe the test was dependant on that exact file :) it's all good =)